fcrepo-fixity/src/test/java/org/fcrepo/camel/fixity/integration/RouteIT.java

```diff
@@ -100,20 +98,25 @@ protected Properties useOverridePropertiesWithPropertiesComponent() {
     @Override
     protected void addServicesOnStartup(final Map<String, KeyValueHolder<Object, Dictionary>> services) {
         final String jmsPort = System.getProperty("fcrepo.dynamic.jms.port", "61616");
+        final String webPort = System.getProperty("fcrepo.dynamic.test.port", "8080");
         final ActiveMQComponent component = new ActiveMQComponent();
 
         component.setBrokerURL("tcp://localhost:" + jmsPort);
         component.setExposeAllQueues(true);
 
-        services.put("broker", asService(component, "osgi.jndi.service.name", "fcrepoqueue"));
+        final FcrepoComponent fcrepo = new FcrepoComponent();
+        fcrepo.setBaseUrl("http://localhost:" + webPort + "/fcrepo/rest");
+
+        services.put("broker", asService(component, "osgi.jndi.service.name", "fcrepo/Broker"));
+        services.put("fcrepo", asService(fcrepo, "osgi.jndi.service.name", "fcrepo/Camel"));
     }
 
     @Test
     public void testFixityOnBinary() throws Exception {
         final String webPort = System.getProperty("fcrepo.dynamic.test.port", "8080");
         final String jmsPort = System.getProperty("fcrepo.dynamic.jms.port", "61616");
         final String path = fullPath.replaceFirst("http://localhost:[0-9]+/fcrepo/rest", "");
-        final String fcrepoEndpoint = "mock:fcrepo:localhost:" + webPort + "/fcrepo/rest";
+        final String fcrepoEndpoint = "mock:fcrepo:http://localhost:8080/fcrepo/rest";
```

Comment:
Why hardcode 8080 here versus use `webport`? Is the `fcrepoEndpoint` different from the `FCREPO_BASE_URL`?